.vscode/settings.json

```diff
@@ -18,5 +18,5 @@
         "./bazel-out/_coverage/_coverage_report.dat"
     ],
     "python.formatting.provider": "yapf",
-    "editor.formatOnSave": true
-}
+    "editor.formatOnSave": false
```

Comment:
I don't think this change is necessary.

Can you add this dir `benchmarks` to https://github.com/pytorch/xla/blob/175e86b82bc0e89a875903e424385ae2adce1bc7/.github/workflows/lintercheck.yml#L69 so CI change check the linter? We usually run `yapf -i -r *.py test/ scripts/ torch_xla/ benchmarks/` to fix the linter

Reply:
Thanks!